File: HarvesterBack/Harvester/Harvester.Application/Dtos/FieldDto.cs
using Harvester.Application.Dtos;
using NetTopologySuite.Geometries;

namespace Harvester.Application
{
    public class FieldDto
    {
        public int Id { get; set; }
        public string IdentifierName { get; set; }
        public string? CommonName { get; set; }

        public decimal AreaHectares { get; set; }
        public decimal TerrainCoeff { get; set; } = 1.0m;
        public decimal ShapeCoeff { get; set; } = 1.0m;
        public string CropType { get; set; }

        public GeoPointDto? CenterPoint { get; set; } = default!;

        public GeoMultiPolygonDto? Boundary { get; set; } = default!;
    }
}

File: HarvesterFront/src/app/core/models/dtos/field-dto.ts
import { GeoMultiPolygonDto } from '../geoJson/geo-multiPolygon-dto';
import { GeoPointDto } from '../geoJson/geo-point-dto';

export interface FieldDto {
  identifierName: string;
  commonName: string;
  areaHectares: number;
  terrainCoeff: number;
  shapeCoeff: number;
  cropType: string;
  centerPoint: GeoPointDto;
  boundary: GeoMultiPolygonDto;
}
